doc/radosgw/frontends.rst

```diff
@@ -86,6 +86,14 @@ Options
 :Type: Integer
 :Default: None
 
+``request_timeout_ms``
+
+:Description: The amount of time in milliseconds that Beast will wait
+              for more incoming data or outgoing data before giving up.
```

Comment:
we should have special handling for request_timeout_ms=0 to disable timeouts, rather than time out immediately

Reply:
I will add conditions for it.
Second thing I will start to change async_write to async_write_some in write_data for GET requests.

Reply:
Fixed